source/_posts/2015-10-17-headless-firefox-in-ubuntu-on-virtual-box-for-cucumber-testing.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Headless firefox in ubuntu on virtual box for cucumber testing"
+date: 2015-10-17 14:12:56 +0200
+comments: true
+author: Ivica Lakatoš
+categories: 
+  - Ruby and Rails
+  - Testing
+tags: 
+  - ruby
+  - rails
+  - vagrant
+  - virtual box
+  - ubuntu
+  - cucumber
+  - selenium-webdriver
+published: true
+---
+
+If you use [Vagrant](http://www.vagrantup.com/downloads.html) and [VirtualBox](https://www.virtualbox.org/) (e.g. some Ubuntu package) to build your Rails app and you want to test it with cucumber scenarios, this is right post for you. 
+
+Everything goes well until you do not need `@javascript` flag for your cucumber scenario. `@javascript` uses a javascript-aware system to process web requests (e.g., Selenium) instead of the default (non-javascript-aware) webrat browser.
+
```

Comment:
@Ikera for longer posts put "more" button at some point in the post. You can decide where, just add tag like I did here https://github.com/orangeiceberg/icebergist.com/commit/15be2e172774b9fb95bac46aed84ecbd8b192699 and the post will be split in two (part which will be displayed on any post lists and the rest of the post).